src/data/senoppfolging/useSenoppfolgingSvarQuery.ts

```diff
@@ -11,7 +12,8 @@ export const senOppfolgingSvarQueryKeys = {
 export const useSenOppfolgingSvarQuery = () => {
   const fnr = useValgtPersonident();
   const path = `${MEROPPFOLGING_BACKEND_ROOT}/senoppfolging/formresponse`;
-  const fetchSenOppfolgingSvar = () => get<string>(path, fnr); // TODO: Fix correct DTO type when api is ready
+  const fetchSenOppfolgingSvar = () =>
+    get<SenOppfolgingFormResponseDTOV2>(path, fnr);
```

Comment:
Hva tenker du er forskjellen på en`fetch…` og `get…`? 😊

Reply:
Jeg bare endret på DTOen, så tenkte ikke over at det sto det ene eller det andre. Men kan godt kalle den `getSenOppfolgingSvar` i stedet. `fetch` er kanskje mer en henvisning til [fetch-APIet](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) som axios kaller under the hood.